stubs/Application/Routers/RouteList.stub

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Nette\Routing;
+
+interface Router
+{
+
+}
+
+namespace Nette\Application\Routers;
+
+/**
+ * @phpstan-implements \IteratorAggregate<int, \Nette\Routing\Router>
+ * @phpstan-implements \ArrayAccess<int, \Nette\Routing\Router>
+ */
+class RouteList extends \Nette\Routing\RouteList implements \Nette\Application\IRouter, \ArrayAccess, \Countable, \IteratorAggregate
```

Comment:
Please add these in separate files. I don't like multiple namespace in a single file.